LocalPayment/src/test/java/com/braintreepayments/api/LocalPaymentClientUnitTest.java

```diff
@@ -213,8 +213,8 @@ public void startPayment_success_sendsAnalyticsEvents() {
         LocalPaymentRequest request = getIdealLocalPaymentRequest();
         sut.startPayment(request, localPaymentStartCallback);
 
-        verify(braintreeClient).sendAnalyticsEvent("ideal.local-payment.start-payment.selected");
-        verify(braintreeClient).sendAnalyticsEvent("ideal.local-payment.create.succeeded");
+        verify(braintreeClient).sendAnalyticsEvent("ideal.local-payment.start-payment.selected", null);
+        verify(braintreeClient).sendAnalyticsEvent("ideal.local-payment.create.succeeded", null);
```

Comment:
Do we want to test that the `payPalContextId` values are being sent as expected here?

Reply:
I added couple of tests here [eb61e1f](https://github.com/braintree/braintree_android/pull/934/commits/eb61e1fe13dcb530f8bae8d534af8f2b38225eff)